symplyphysics/laws/conservation/abbe_invariant_of_two_optical_environments_is_constant.py

```diff
@@ -0,0 +1,64 @@
+from sympy import Eq, solve, S
+from symplyphysics import (Symbol, units, print_expression, Quantity,
+                           validate_input, validate_output, dimensionless,
+                           convert_to)
+
+# Description
+## The Abbe's invariant connects the front and back segments S and S',
+## allowing one of them to be determined if the second one is known
+
+# Law: n * ((1 / a) - (1 / R)) = n' * ((1 / b) - (1 / R))
+# Where:
+## n - refraction index in optical environment
+## n' - refraction index in lens
```

Comment:
Somehow you limit it to lens, but this law is about any surface with constant curvature radius, between 2 optical environments.